include/oneapi/dpl/internal/distributed_ranges_impl/detail/enumerate.hpp

```diff
@@ -0,0 +1,97 @@
+// -*- C++ -*-
+//===----------------------------------------------------------------------===//
+//
+// Copyright (C) Intel Corporation
+//
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+// This file incorporates work covered by the following copyright and permission
+// notice:
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+//
+//===----------------------------------------------------------------------===//
+
+#pragma once
+
+#include <oneapi/dpl/internal/distributed_ranges_impl/detail/ranges_shim.hpp>
```

Comment:
Our recommended way to include other internal files is to use their relative path and quotes `""` instead of angle brackets `<>`. I.e, here it would be `#include "ranges_shim.hpp"`. One advantage is that changing the directory names and structure within the project is much easier when it's not hard-coded in every include directive.

Reply:
#1728

Reply:
Applied in https://github.com/oneapi-src/oneDPL/pull/1744 and https://github.com/oneapi-src/oneDPL/pull/1728
Resolving.